wire-tests/src/commonTest/proto-kotlin/com/squareup/wire/protos/kotlin/alltypes/AllTypes.kt

```diff
@@ -2686,73 +2698,73 @@ public class AllTypes(
               if (array_int32 == null) {
                 array_int32 = IntArrayList.forDecoding(reader.nextFieldMinLengthInBytes(), 1)
               }
-              array_int32!!.add(com.squareup.wire.`internal`.decodePrimitive_int32(reader))
+              array_int32!!.add(decodePrimitive_int32(reader))
             }
             602 -> {
               if (array_uint32 == null) {
                 array_uint32 = IntArrayList.forDecoding(reader.nextFieldMinLengthInBytes(), 1)
               }
-              array_uint32!!.add(com.squareup.wire.`internal`.decodePrimitive_uint32(reader))
+              array_uint32!!.add(decodePrimitive_uint32(reader))
             }
             603 -> {
               if (array_sint32 == null) {
                 array_sint32 = IntArrayList.forDecoding(reader.nextFieldMinLengthInBytes(), 1)
               }
-              array_sint32!!.add(com.squareup.wire.`internal`.decodePrimitive_sint32(reader))
+              array_sint32!!.add(decodePrimitive_sint32(reader))
             }
             604 -> {
               if (array_fixed32 == null) {
                 array_fixed32 = IntArrayList.forDecoding(reader.nextFieldMinLengthInBytes(), 4)
               }
-              array_fixed32!!.add(com.squareup.wire.`internal`.decodePrimitive_fixed32(reader))
+              array_fixed32!!.add(decodePrimitive_fixed32(reader))
             }
             605 -> {
               if (array_sfixed32 == null) {
                 array_sfixed32 = IntArrayList.forDecoding(reader.nextFieldMinLengthInBytes(), 4)
               }
-              array_sfixed32!!.add(com.squareup.wire.`internal`.decodePrimitive_sfixed32(reader))
+              array_sfixed32!!.add(decodePrimitive_sfixed32(reader))
             }
             606 -> {
               if (array_int64 == null) {
                 array_int64 = LongArrayList.forDecoding(reader.nextFieldMinLengthInBytes(), 1)
               }
-              array_int64!!.add(com.squareup.wire.`internal`.decodePrimitive_int64(reader))
```

Comment:
These becoming not-fully-qualified is alarming

Reply:
.... ahhhh it’s cause I stopped doing unnecessary toString() on 'em. Never mind then!